src/ui/templates/components/personal-details-input.njk

```diff
@@ -0,0 +1,43 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% macro render(params) %}
+
+  {% set class = params.class %}
+  {% set fieldId = params.fieldId %}
+  {% set label = params.label %}
+  {% set labelClass = params.labelClass %}
+  {% set hint = params.hint %}
+  {% set submittedValues = params.submittedValues %}
+  {% set applicationAnswers = params.submittedAnswers %}
+  {% set validationErrors = params.validationErrors %}
+
+  {{ govukInput({
+    id: fieldId,
+    name: fieldId,
+    value: submittedValues[fieldId] or applicationAnswers[FIELDS.FIRST_NAME.ID],
```

Comment:
Not a blocker - typo here with `FIELDS.FIRST_NAME.ID`

Reply:
whoops, will fix in next PR